core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/PartitionInfo.scala

```diff
@@ -72,6 +72,11 @@ final case class RangePartition(rangeAttributeNames: List[String], rangeMin: Lon
   */
 final case class SinglePartition() extends PartitionInfo {}
 
+/**
+  * Represent the input stream is not partitioned and all data are on a single node.
+  */
```

Comment:
Please update this comment so that it is differentiated from the `SinglePartition`.

Reply:
Fixed.